include/iocore/cache/CacheVC.h

```diff
@@ -49,6 +49,8 @@
 #include <cstdint>
 
 struct Stripe;
+struct Vol;
```

Comment:
Is there still a `struct Vol`?

Reply:
I probably resolved conflict in a wrong way.